lib/fluent/plugin/out_aws-elasticsearch-service.rb

```diff
@@ -111,6 +113,12 @@ def calback.inspect
       calback
     end
 
+    def sts_credentials_region(opts)
+      opts[:sts_credentials_region] || opts[:region]
+    end
+
+
+
```

Comment:
Could you remove these newlines?
In Ruby style, using one newline between the functions is better.

Reply:
Removed